README.md

```diff
@@ -1 +1,163 @@
-# Group-Recipe-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
```

Comment:
- [ ] There are no instructions on how to run the tests Please add that so once would know how to do it,

Reply:
Following commit for the solution:

https://github.com/Linushaddai99/Group-Recipe-app/pull/33/commits/76662abd273e22259bbcbd901be8bb545e01912d